torch/nn/quantized/dynamic/modules/linear.py

```diff
@@ -28,18 +28,25 @@ class Linear(nnq.Linear):
         torch.Size([128, 30])
     """
 
+    _version = 4
```

Comment:
just for my own learning, the previous version was 3 even though it wasn't specified in this file?

Reply:
Yeah, dynamic Linear inherits from nn.quantized.Linear which currently has the version set at 3. So I decided to keep an independent version in this file. In the future if we bump the version of nnq.Linear to 4 or higher, this should still work since we check for `self.version < 4` in forward to call the older API.
Alternately I could bump the version in nnq.Linear to 4. But that would involve more changes to that file and might be a bit confusing. Let me know if you think that is better.

Reply:
makes sense.  Maybe we can add a comment that this version is different from the parent class, since it's easy to miss?